src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/UnnecessaryParenthesesCheck.java

```diff
@@ -552,10 +552,16 @@ else if (isSurrounded(ast) && unnecessaryParenAroundOperators(ast)) {
      *         parentheses.
      */
     private static boolean isSurrounded(DetailAST ast) {
```

Comment:
Please update javadoc to be more simple; we do not need to know implementation details of a method. We really only need the first sentence:

```
Tests if the given {@code DetailAST} is surrounded by parentheses.

```

Reply:
done